src/assets/scripts/client/navigationLibrary/NavigationLibrary.js

```diff
@@ -316,4 +323,45 @@ export default class NavigationLibrary {
             flightPhase === FLIGHT_PHASE.TAXI ||
             flightPhase === FLIGHT_PHASE.WAITING;
     }
+
+    /**
+     * Check all fixes used in procedures, and gather a list of any fixes that are
+     * not defined in the `fixes` section of the airport file, then sort and print
+     * that list to the console.
+     *
+     * @for NavigationLibrary
+     * @method showConsoleWarningForUndefinedFixes
+     */
+    showConsoleWarningForUndefinedFixes() {
+        const allFixNames = this._getAllFixNamesInUse();
+        const missingFixes = allFixNames.filter((fix) => !FixCollection.findFixByName(fix));
+
+        if (missingFixes.length < 1) {
+            return;
+        }
+
+        console.warn(`The following fixes have yet to be defined in the "fixes" section: ${missingFixes}`);
+    }
+
+    /**
+     * Gathers a unique, sorted list of all fixes used in all known procedures
+     *
+     * @for NavigationLibrary
+     * @method _getAllFixNamesInUse
+     * @return {array<string>} ['fixxa', 'fixxb', 'fixxc', ...]
+     * @private
+     */
+    _getAllFixNamesInUse() {
+        const allFixNames = [
+            ...this.sidCollection.getAllFixNamesInUse(),
+            ...this.starCollection.getAllFixNamesInUse()
+        ];
+        const uniqueFixNames = _uniq(allFixNames);
+        const allNonVectorFixes = _filter(uniqueFixNames, (fix) => {
+            return fix.indexOf(VECTOR_WAYPOINT_PREFIX) === INVALID_INDEX;
```

Comment:
they're both valid, but yes, that was the question. I'll move the `);` back onto the end of the same line.